Presentation.Web/Controllers/API/V2/Common/Mapping/OrganizationMapper.cs

```diff
@@ -0,0 +1,176 @@
+        {
+            return new()
+            {
+                Email = dataResponsible.Email,
+                Id = dataResponsible.Id,
+                Cvr = dataResponsible.Cvr,
+                Name = dataResponsible.Name,
+                Address = dataResponsible.Adress,
+                Phone = dataResponsible.Phone,
+            };
+        }
+
+        private static DataProtectionAdvisorResponseDTO ToDataProtectionAdvisorDTO(DataProtectionAdvisor dataProtectionAdvisor)
+        {
+            return new()
+            {
+                Email = dataProtectionAdvisor.Email,
+                Id = dataProtectionAdvisor.Id,
+                Cvr = dataProtectionAdvisor.Cvr,
+                Name = dataProtectionAdvisor.Name,
+                Address = dataProtectionAdvisor.Adress,
+                Phone = dataProtectionAdvisor.Phone,
+            };
+        }
+
+        public OrganizationMasterDataUpdateParameters ToMasterDataUpdateParameters(OrganizationMasterDataRequestDTO dto)
+        {
+            return new()
+            {
+                Cvr = OptionalValueChange<string>.With(dto.Cvr),
+                Email = OptionalValueChange<string>.With(dto.Email),
+                Address = OptionalValueChange<string>.With(dto.Address),
+                Phone = OptionalValueChange<string>.With(dto.Phone),
+            };
+        }
+
+        public OrganizationMasterDataRolesUpdateParameters ToMasterDataRolesUpdateParameters(Guid organizationUuid,
+            OrganizationMasterDataRolesRequestDTO dto)
+        {
+            var contactPersonDto = dto.ContactPerson;
+            var contactPersonParameters = ToContactPersonUpdateParameters(contactPersonDto);
+
+            var dataResponsibleDto = dto.DataResponsible;
+            var dataResponsibleParameters = ToDataResponsibleUpdateParameters(dataResponsibleDto);
+
+            var dataProtectionAdvisorDto = dto.DataProtectionAdvisor;
+            var dataProtectionAdvisorParameters = ToDataProtectionAdvisorUpdateParameters(dataProtectionAdvisorDto);
+
+            return new()
+            {
+                OrganizationUuid = organizationUuid,
+                ContactPerson = contactPersonParameters,
+                DataResponsible = dataResponsibleParameters,
+                DataProtectionAdvisor = dataProtectionAdvisorParameters
+            };
+        }
+
+        public OrganizationMasterDataResponseDTO ToMasterDataDTO(OrganizationMasterData masterData)
+        {
+            return new()
+            {
+                Cvr = masterData.Cvr,
+                Address = masterData.Address,
+                Email = masterData.Email,
+                Phone = masterData.Phone,
+                Uuid = masterData.Uuid,
+                Name = masterData.Name
+            };
+        }
+
+        private static Maybe<ContactPersonUpdateParameters> ToContactPersonUpdateParameters(ContactPersonRequestDTO dto)
```

Comment:
Check the existing WriteMappers for how to perform the mapping. You have to check for changes, and check if a field must update + if the default value should be enforced (so it always has a value). Usually value is enforced in POST methods, and optional in PUT and PATCH. It should look like this (example from ItSystemUsageWriteModelMapper) 
<img width="651" alt="image" src="https://github.com/user-attachments/assets/598b2aa2-206f-4f6b-a732-de2682dd410a">